pretix_eth/txn_check.py

```diff
@@ -0,0 +1,75 @@
+import codecs
+import requests
+
+def json_rpc_request(method, params):
+    try:
+        payload = { "id": 1337, "jsonrpc": "2.0", "method": method, "params": params }
```

Comment:
I'll come through and cleanup this file with a pattern that lets us test it in a follow up PR so assuming this code works right now I don't see any need to do much cleanup on it since I'll rework most of it.